app/controllers/invitations_to_organization_controller.rb

```diff
@@ -0,0 +1,49 @@
+class InvitationsToOrganizationController < ApplicationController
```

Comment:
@dissident I think it will be better to name it `OrganiztaionInvitationsContoller`. Same for model